### PR TITLE
feat(medusa): seed command can create product categories

### PR DESCRIPTION
**Pull request by riqwan**
What:

- The medusa seed command can create nested product categories

RESOLVES CORE-1274

### CODE REVIEW

**Review by fPolic — APPROVED** [review]
LGTM!

**Review by olivermrbl — APPROVED** [review]

File: packages/medusa/src/commands/seed.ts [hunk 1]
```diff
@@ -202,6 +210,33 @@ const seed = async function ({ directory, migrate, seedFile }: SeedOptions) {
         }
       }
     }
+
+    const createProductCategory = async (
```
[the remark this answers is not in the document]
Comment (riqwan) [reply packages/medusa/src/commands/seed.ts:214]: I've added to the staging repo - https://github.com/medusajs/medusa-staging/pull/20

Should we add the same thing in there or skim it down?

[the remark this answers is not in the document]
Comment (olivermrbl) [reply packages/medusa/src/commands/seed.ts:214]: Holy smokes - this is super nice! 

Though, maybe skim it down to 5-6 categories with a couple of children each?

[the remark this answers is not in the document]
Comment (riqwan) [reply packages/medusa/src/commands/seed.ts:214]: Haha, power of writing a script and some scraping. :D 

I'll add a skimmed version to the starter.

**Pull request merged**